scripts/prepare-rolling-demo.sh

```diff
@@ -120,6 +120,15 @@ kubectl create secret generic "$SECRET_NAME" \
     --dry-run=client -o yaml | kubectl apply --filename - --overwrite=true >/dev/null
 echo "OK"
 
+SECRET_NAME="ai-rh-developer-hub-env"
+echo -n "* $SECRET_NAME secret: "
+kubectl create secret generic "$SECRET_NAME" \
+    --namespace="$RHDH_NAMESPACE" \
+    --from-literal=NODE_TLS_REJECT_UNAUTHORIZED="0" \
+    --from-literal=RHDH_TOKEN="$ADMIN_TOKEN"
```

Comment:
I wonder if over time we should use a k8s serviceaccount / secret to populate our backstage administrative static token vs. the env var approach currently done by the prepare-rolling-demo.sh

I suspect a customer, if they chose to do static tokens vs. jwt tokens for real, would employ that approach

WDYT @thepetk @johnmcollier ?

if you all agree, do we want to take that on with this PR, or a f/up PR?

if agree

Reply:
Yeah +1. Just to confirm, we would like here the `RHDH_TOKEN` value to be a token from a k8s service account (e.g. the already created `k8s-sa`).

Reply:
I think we should have a separate SA / secret / token than the one we use for the bridge's containers, but otherwise, yes, you understand the ask

Reply:
I have added a new sa creation in the prepare-rolling-demo.sh script and I pass the token for this SA to be the RHDH (admin) token.